wiki/en/Include-Server-Commands.md

```diff
@@ -1,19 +1,19 @@
-- `-e`  or `--directoryserver`  Register the server on a directory (e.g. to set its genre (see also `-o`)). See [server types](#server-types) for further information.
-- `--directoryfile` Enable directory to remember registered servers even if the directory is restarted. You'll need to specify a path to a writable file location in which server information is stored.
-- `-f`  or `--listfilter`     Whitelist servers registering on the server list, format `ip address 1[;ip address 2]` Directory Servers only. See [server types](#server-types)  
+- `-d`  or `--discononquit`   Disconnect all clients on quit. Normally, when a Server is stopped or restarted, any clients that have not used their "Disconnect" buttons will re-establish connection when the Server comes back up again. Using this option forces clients to manually re-establish their connections to the Server.  
+- `-e`  or `--directoryserver`  Register the Server on a Directory (e.g. to set its genre (see also `-o`)). See [Server Types](#server-types) for further information.
+- `--directoryfile` Remember registered Servers even if the Directory is restarted. Directory Servers only. See [this guide](Custom-Directories) for further information. 
+- `-f`  or `--listfilter`     Whitelist Servers registering on the Server list, format `ip address 1[;ip address 2]` Directory Servers only. 
 -  `-F`  or `--fastupdate`     Reduces latency if clients connect with "Enable Small Network Buffers" option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled clients.
 -  `-l`  or `--log`            Enable logging, set path and file name                                                                    
 - `-L`  or `--licence`        Show an agreement window before users can connect
 - `-m`  or `--htmlstatus`     Enable HTML status file, set path and file name
-- `-o`  or `--serverinfo`     Location details in the format:  `[name];[city];[locale value]` (see [values](https://doc.qt.io/qt-5/qlocale.html#Country-enum)) Public servers only
+- `-o`  or `--serverinfo`     Location details in the format:  `[name];[city];[locale value]` (see [values](https://doc.qt.io/qt-5/qlocale.html#Country-enum)) Registered Servers only
 - `-P`  or `--delaypan`       Start with delay panning enabled See [notes](#delay-panning)
 - `-R`  or `--recording`      Include a writeable path where the files should be stored (in quotes if needed).  See [Options](#options).  
 - `--norecord`       Disable recording when enabled by default by `-R`                                                 
-- `-s` or `--server` Start in server mode
+- `-s` or `--server` Start in Server mode
 - `--serverbindip`  Specify the IP address to bind to              
 -  `-T` or `--multithreading`  Use multithreading to make better use of multi-core CPUs to support more clients
 -  `-u` or `--numchannels`   Maximum number of channels (clients)
 -  `-w` or `--welcomemessage`  Welcome message on connect
```

Comment:
This can be given as a string or filename (if the value given is the name of a file that can be read, it is used as such, otherwise as a string).  The welcome message can contain HTML.

Reply:
What's "this" refer to? BTW the PR here is just for the server vocab change really. 

Also this reminds me that if we change anything here we also need to change it in that damn man page.

Reply:
I was providing additional text for the description of the option.  So, in full:

> Welcome message on connect.  This can be given as a string or filename (if the value given is the name of a file that can be read, it is used as such, otherwise as a string). The welcome message can contain HTML.